assets/src/components/SelectCard.js

```diff
@@ -235,11 +235,11 @@ const SelectCard = props => {
       </CardContent>
     )
 
-    return <Root>{cardImage}{cardContent}</Root>
+    return <>{cardImage}{cardContent}</>
   }
```

Comment:
This is nice and simple fix. I pulled the latest and things are looking good with this commit.